libs/utils/utils.js

```diff
@@ -74,7 +74,7 @@ const MILO_BLOCKS = [
   'reading-time',
 ];
 const AUTO_BLOCKS = [
-  { adobetv: 'https://video.tv.adobe.com' },
+  { adobetv: 'tv.adobe.com' },
```

Comment:
👍 

I didn't see a test for this in any of your pages. So, I added [a .mp4 MPC video](https://mwpw-135720-marquee-video--milo--adobecom.hlx.page/drafts/rclayton/video/videos) to a sample page to verify